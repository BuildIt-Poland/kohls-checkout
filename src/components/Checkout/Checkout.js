import React from 'react';
import { Route } from 'react-router-dom';
import styled from 'styled-components';

import CheckoutDelivery from '../CheckoutDelivery';
import CheckoutPayment from '../CheckoutPayment';
import CheckoutReview from '../CheckoutReview';
import Navigation from '../Navigation';
import { CHECKOUT_DELIVERY, CHECKOUT_PAYMENT, CHECKOUT_REVIEW, CHECKOUT } from '../../constants/routes';

const Wrapper = styled.section`
  margin-top: 100px;
`;

function Checkout() {
  return (
    <>
      <Route path={CHECKOUT} component={Navigation} />
      <Wrapper>
        <Route path={CHECKOUT_DELIVERY} component={CheckoutDelivery} />
        <Route path={CHECKOUT_PAYMENT} component={CheckoutPayment} />
        <Route path={CHECKOUT_REVIEW} component={CheckoutReview} />
      </Wrapper>
    </>
  );
}

export default Checkout;
