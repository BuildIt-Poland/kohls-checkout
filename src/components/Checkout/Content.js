import styled from 'styled-components';

import { spacingLarge } from '../../styles/designTokens';
import ContentBase from '../Content';

const Content = styled(ContentBase)`
  padding-top: ${spacingLarge};
  padding-bottom: ${spacingLarge};
`;

export default Content;
