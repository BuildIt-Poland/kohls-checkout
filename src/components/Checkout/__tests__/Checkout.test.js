import React from 'react';
import { create } from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';

import Checkout from '../Checkout';

describe('COMPONENT - Checkout', () => {
  it('should render correctly delivery route', () => {
    const component = create(
      <MemoryRouter initialEntries={['/checkout/delivery']} initialIndex={1}>
        <Checkout />
      </MemoryRouter>
    );

    expect(component.toJSON()).toMatchSnapshot();
  });

  it('should render correctly payment route', () => {
    const component = create(
      <MemoryRouter initialEntries={['/checkout/payment']} initialIndex={1}>
        <Checkout />
      </MemoryRouter>
    );

    expect(component.toJSON()).toMatchSnapshot();
  });

  it('should render correctly review route', () => {
    const component = create(
      <MemoryRouter initialEntries={['/checkout/review']} initialIndex={1}>
        <Checkout />
      </MemoryRouter>
    );

    expect(component.toJSON()).toMatchSnapshot();
  });
});
