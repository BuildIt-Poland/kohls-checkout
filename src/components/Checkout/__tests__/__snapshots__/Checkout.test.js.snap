// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - Checkout renders correctly for "/delivery" route 1`] = `
Array [
  .c3 {
  background: #ffffff;
  color: #222222;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  font-size: 1.8rem;
  font-weight: 700;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  width: 100%;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 100%;
  -webkit-text-decoration: none;
  text-decoration: none;
  border-left: 1px solid #b0b0b0;
}

.c2:last-child {
  border-right: 1px solid #b0b0b0;
}

.c0 {
  width: 100%;
  border-bottom: 1px solid #b0b0b0;
}

.c1 {
  max-width: 100rem;
  padding: 0 2rem;
  margin: 0 auto;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 6rem;
}

.c4 {
  font-weight: 400;
  padding-right: 0.4rem;
}

@media (max-width:48em) {
  .c2:first-child {
    border-left: 0;
  }

  .c2:last-child {
    border-right: 0;
  }
}

@media (max-width:48em) {
  .c1 {
    padding: 0;
  }
}

<div
    className="c0"
  >
    <nav
      className="c1"
    >
      <a
        className="c2"
        data-testid="navigation-link"
        href="/checkout/delivery"
        onClick={[Function]}
      >
        <span
          className="c3"
        >
          <span
            className="c4"
          >
            1.
          </span>
          Delivery
        </span>
      </a>
      <a
        className="c2"
        data-testid="navigation-link"
        href="/checkout/payment"
        onClick={[Function]}
      >
        <span
          className="c3"
        >
          <span
            className="c4"
          >
            2.
          </span>
          Payment
        </span>
      </a>
      <a
        className="c2"
        data-testid="navigation-link"
        href="/checkout/review"
        onClick={[Function]}
      >
        <span
          className="c3"
        >
          <span
            className="c4"
          >
            3.
          </span>
          Review
        </span>
      </a>
    </nav>
  </div>,
  .c0 {
  padding-top: 3rem;
  padding-bottom: 12rem;
}

.c2 {
  margin: 0;
  margin-bottom: 2rem;
  font-size: 3rem;
  font-weight: 700;
  line-height: 1;
}

.c1 {
  max-width: 100rem;
  padding: 0 2rem;
  margin: 0 auto;
}

.c4 {
  border: 0;
  background: none;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  min-width: 2rem;
  min-height: 2rem;
  cursor: pointer;
}

.c4 svg {
  display: block;
}

.c6 {
  background: #00a09f;
  color: #ffffff;
  display: block;
  font-size: 2.4rem;
  font-weight: 700;
  padding: 1.2rem;
  text-align: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  max-width: 96rem;
  margin: 0 auto;
}

.c5 {
  bottom: 0;
  left: 0;
  padding: 1rem;
  position: fixed;
  width: 100%;
  background: #ffffff;
  border-top: 1px solid #b0b0b0;
  z-index: 6000;
}

.c3 {
  font-size: 2.4rem;
  text-align: left;
  margin: 1rem 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

<article
    className="c0"
  >
    <section
      className="c1"
    >
      <h1
        className="c2"
      >
        Delivery
      </h1>
      <section>
        <h2
          className="c3"
        >
          Shipping Address
          <button
            className="c4"
            data-testid="modal-trigger"
            onClick={[Function]}
          >
            <svg
              height="18"
              width="18"
              xmlns="http://www.w3.org/2000/svg"
            >
              <circle
                cx="9"
                cy="9"
                fill="#222"
                r="9"
              />
              <path
                d="M9.833 6.241H8.167V4.724h1.666v1.517zM8.167 7.49h1.666v5.786H8.167V7.49z"
                fill="#fff"
              />
            </svg>
          </button>
        </h2>
      </section>
    </section>
    <div
      className="c5"
    >
      <a
        className="c6"
        data-testid="next-step-button"
        href="/checkout/payment"
        onClick={[Function]}
      >
        Continue to Payment
      </a>
    </div>
  </article>,
]
`;

exports[`COMPONENT - Checkout renders correctly for "/payment" route 1`] = `
Array [
  .c3 {
  background: #ffffff;
  color: #222222;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  font-size: 1.8rem;
  font-weight: 700;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  width: 100%;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 100%;
  -webkit-text-decoration: none;
  text-decoration: none;
  border-left: 1px solid #b0b0b0;
}

.c2:last-child {
  border-right: 1px solid #b0b0b0;
}

.c0 {
  width: 100%;
  border-bottom: 1px solid #b0b0b0;
}

.c1 {
  max-width: 100rem;
  padding: 0 2rem;
  margin: 0 auto;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 6rem;
}

.c4 {
  font-weight: 400;
  padding-right: 0.4rem;
}

@media (max-width:48em) {
  .c2:first-child {
    border-left: 0;
  }

  .c2:last-child {
    border-right: 0;
  }
}

@media (max-width:48em) {
  .c1 {
    padding: 0;
  }
}

<div
    className="c0"
  >
    <nav
      className="c1"
    >
      <a
        className="c2"
        data-testid="navigation-link"
        href="/checkout/delivery"
        onClick={[Function]}
      >
        <span
          className="c3"
        >
          <span
            className="c4"
          >
            1.
          </span>
          Delivery
        </span>
      </a>
      <a
        className="c2"
        data-testid="navigation-link"
        href="/checkout/payment"
        onClick={[Function]}
      >
        <span
          className="c3"
        >
          <span
            className="c4"
          >
            2.
          </span>
          Payment
        </span>
      </a>
      <a
        className="c2"
        data-testid="navigation-link"
        href="/checkout/review"
        onClick={[Function]}
      >
        <span
          className="c3"
        >
          <span
            className="c4"
          >
            3.
          </span>
          Review
        </span>
      </a>
    </nav>
  </div>,
  .c0 {
  padding-top: 3rem;
  padding-bottom: 12rem;
}

.c2 {
  margin: 0;
  margin-bottom: 2rem;
  font-size: 3rem;
  font-weight: 700;
  line-height: 1;
}

.c1 {
  max-width: 100rem;
  padding: 0 2rem;
  margin: 0 auto;
}

.c10 {
  background: #00a09f;
  color: #ffffff;
  display: block;
  font-size: 2.4rem;
  font-weight: 700;
  padding: 1.2rem;
  text-align: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  max-width: 96rem;
  margin: 0 auto;
}

.c9 {
  bottom: 0;
  left: 0;
  padding: 1rem;
  position: fixed;
  width: 100%;
  background: #ffffff;
  border-top: 1px solid #b0b0b0;
  z-index: 6000;
}

.c4 {
  font-size: 2.4rem;
  text-align: left;
  margin: 1rem 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c5 {
  font-size: 1.8rem;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-weight: 400;
  line-height: 1.4;
  margin: 0;
  margin-bottom: 1rem;
  color: #222222;
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c5:last-child {
  margin-bottom: 0;
}

.c6 {
  font-size: 1.8rem;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-weight: 400;
  line-height: 1.4;
  margin: 0;
  margin-bottom: 1rem;
  color: #008484;
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c6:last-child {
  margin-bottom: 0;
}

.c8 {
  font-weight: 700;
}

.c7 {
  font-size: 1.8rem;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-weight: 400;
  line-height: 1.4;
  margin: 0;
  margin-bottom: 1rem;
  color: #222222;
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  border-top: 1px solid #b0b0b0;
  padding-top: 2rem;
  margin-top: 2rem;
}

.c7:last-child {
  margin-bottom: 0;
}

.c3 {
  border-top: 1px solid #b0b0b0;
  padding-top: 2rem;
}

<article
    className="c0"
  >
    <section
      className="c1"
    >
      <h1
        className="c2"
      >
        Payment
      </h1>
    </section>
    <section
      className="c3"
    >
      <section
        className="c1"
      >
        <h2
          className="c4"
        >
          Order Summary
        </h2>
        <p
          className="c5"
        >
          <span>
            Subtotal
          </span>
          <span>
            $125.00
          </span>
        </p>
        <p
          className="c6"
        >
          <span>
            Kohl's Cash & Discounts
          </span>
          <span>
            -$18.75
          </span>
        </p>
        <p
          className="c5"
        >
          <span>
            Shipping
          </span>
          <strong>
            FREE
          </strong>
        </p>
        <p
          className="c5"
        >
          <span>
            Tax
          </span>
          <span>
            $6.38
          </span>
        </p>
        <p
          className="c7"
        >
          <span
            className="c8"
          >
            Total
          </span>
          <span
            className="c8"
          >
            $
            122.7
          </span>
        </p>
        <p
          className="c5"
        >
          <span>
            Your Savings
          </span>
          <span>
            $122.70
          </span>
        </p>
      </section>
    </section>
    <div
      className="c9"
    >
      <a
        className="c10"
        data-testid="next-step-button"
        href="/checkout/review"
        onClick={[Function]}
      >
        Review Order
      </a>
    </div>
  </article>,
]
`;

exports[`COMPONENT - Checkout renders correctly for "/review" route 1`] = `
Array [
  .c3 {
  background: #ffffff;
  color: #222222;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  font-size: 1.8rem;
  font-weight: 700;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  width: 100%;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 100%;
  -webkit-text-decoration: none;
  text-decoration: none;
  border-left: 1px solid #b0b0b0;
}

.c2:last-child {
  border-right: 1px solid #b0b0b0;
}

.c0 {
  width: 100%;
  border-bottom: 1px solid #b0b0b0;
}

.c1 {
  max-width: 100rem;
  padding: 0 2rem;
  margin: 0 auto;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 6rem;
}

.c4 {
  font-weight: 400;
  padding-right: 0.4rem;
}

@media (max-width:48em) {
  .c2:first-child {
    border-left: 0;
  }

  .c2:last-child {
    border-right: 0;
  }
}

@media (max-width:48em) {
  .c1 {
    padding: 0;
  }
}

<div
    className="c0"
  >
    <nav
      className="c1"
    >
      <a
        className="c2"
        data-testid="navigation-link"
        href="/checkout/delivery"
        onClick={[Function]}
      >
        <span
          className="c3"
        >
          <span
            className="c4"
          >
            1.
          </span>
          Delivery
        </span>
      </a>
      <a
        className="c2"
        data-testid="navigation-link"
        href="/checkout/payment"
        onClick={[Function]}
      >
        <span
          className="c3"
        >
          <span
            className="c4"
          >
            2.
          </span>
          Payment
        </span>
      </a>
      <a
        className="c2"
        data-testid="navigation-link"
        href="/checkout/review"
        onClick={[Function]}
      >
        <span
          className="c3"
        >
          <span
            className="c4"
          >
            3.
          </span>
          Review
        </span>
      </a>
    </nav>
  </div>,
  .c0 {
  padding-top: 3rem;
  padding-bottom: 12rem;
}

.c2 {
  margin: 0;
  margin-bottom: 2rem;
  font-size: 3rem;
  font-weight: 700;
  line-height: 1;
}

.c1 {
  max-width: 100rem;
  padding: 0 2rem;
  margin: 0 auto;
}

.c10 {
  border: 0;
  background: none;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  min-width: 2rem;
  min-height: 2rem;
  cursor: pointer;
}

.c10 svg {
  display: block;
}

.c8 {
  font-size: 1.8rem;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-weight: 400;
  line-height: 1.4;
  margin: 0;
  margin-bottom: 1rem;
  color: #222222;
}

.c8:last-child {
  margin-bottom: 0;
}

.c20 {
  background: #00a09f;
  color: #ffffff;
  display: block;
  font-size: 2.4rem;
  font-weight: 700;
  padding: 1.2rem;
  text-align: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  max-width: 96rem;
  margin: 0 auto;
}

.c19 {
  bottom: 0;
  left: 0;
  padding: 1rem;
  position: fixed;
  width: 100%;
  background: #ffffff;
  border-top: 1px solid #b0b0b0;
  z-index: 6000;
}

.c3 {
  font-size: 2.4rem;
  text-align: left;
  margin: 1rem 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c15 {
  font-size: 1.8rem;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-weight: 400;
  line-height: 1.4;
  margin: 0;
  margin-bottom: 1rem;
  color: #222222;
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c15:last-child {
  margin-bottom: 0;
}

.c16 {
  font-size: 1.8rem;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-weight: 400;
  line-height: 1.4;
  margin: 0;
  margin-bottom: 1rem;
  color: #008484;
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c16:last-child {
  margin-bottom: 0;
}

.c18 {
  font-weight: 700;
}

.c17 {
  font-size: 1.8rem;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-weight: 400;
  line-height: 1.4;
  margin: 0;
  margin-bottom: 1rem;
  color: #222222;
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  border-top: 1px solid #b0b0b0;
  padding-top: 2rem;
  margin-top: 2rem;
}

.c17:last-child {
  margin-bottom: 0;
}

.c14 {
  border-top: 1px solid #b0b0b0;
  padding-top: 2rem;
}

.c4 {
  color: #008484;
  font-weight: 700;
  font-size: 1.8rem;
  -webkit-text-decoration: underline;
  text-decoration: underline;
  text-transform: none;
}

.c11 {
  padding: 0;
  font-style: normal;
}

.c12 {
  font-size: 1.8rem;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-weight: 700;
  line-height: 1.4;
  margin: 0;
  margin-bottom: 1rem;
  color: #222222;
  margin: 0;
}

.c12:last-child {
  margin-bottom: 0;
}

.c13 {
  font-size: 1.8rem;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-weight: 400;
  line-height: 1.4;
  margin: 0;
  margin-bottom: 1rem;
  color: #222222;
  margin: 0;
}

.c13:last-child {
  margin-bottom: 0;
}

.c5 {
  display: grid;
  grid-template-columns: auto 1fr;
  grid-gap: 20px;
}

.c7 {
  width: 132px;
}

.c6 {
  grid-column-start: 1;
  grid-row-start: 1;
}

.c9 {
  font-size: 1.8rem;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-weight: 700;
  line-height: 1.4;
  margin: 0;
  margin-bottom: 1rem;
  color: #222222;
  margin-left: 1rem;
}

.c9:last-child {
  margin-bottom: 0;
}

<article
    className="c0"
  >
    <section
      className="c1"
    >
      <h1
        className="c2"
      >
        Review
      </h1>
      <section>
        <h2
          className="c3"
        >
          Shipping Details
          <a
            className="c4"
            href="/checkout/delivery"
            onClick={[Function]}
          >
            Edit
          </a>
        </h2>
      </section>
      <section>
        <h2
          className="c3"
        >
          Payment Information
          <a
            className="c4"
            href="/checkout/payment"
            onClick={[Function]}
          >
            Edit
          </a>
        </h2>
        <h2
          className="c3"
        >
          Payment Method
        </h2>
        <div
          className="c5"
        >
          <div
            className="c6"
          >
            <div
              className="c7"
            >
              <svg
                id="Layer_1"
                viewBox="0 0 66 44"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M3 42c-1.654 0-3-1.346-3-3V3c0-1.654 1.346-3 3-3h60c1.654 0 3 1.346 3 3v36c0 1.654-1.346 3-3 3H3z"
                  fill="#a0a0a0"
                />
                <path
                  d="M63 41H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h60c1.1 0 2 .9 2 2v36c0 1.1-.9 2-2 2z"
                  fill="#fff"
                />
                <path
                  d="M63 41H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h60c1.1 0 2 .9 2 2v36c0 1.1-.9 2-2 2z"
                  fill="#fff"
                />
                <path
                  d="M2.066 34.501h61.867v5.457H2.066z"
                  fill="#f7b600"
                />
                <path
                  d="M2.066 2.049h61.867v5.458H2.066z"
                  fill="#1a1f71"
                />
                <path
                  d="M26.83 13.268l-6.391 15H16.27l-3.145-11.971c-.191-.736-.357-1.007-.937-1.318-.948-.507-2.514-.981-3.891-1.276l.093-.435h6.712c.855 0 1.624.56 1.819 1.528l1.661 8.681 4.103-10.209h4.145zm16.336 10.104c.017-3.959-5.562-4.178-5.525-5.947.012-.538.533-1.11 1.672-1.256.565-.072 2.123-.13 3.89.671l.691-3.182c-.949-.338-2.17-.663-3.689-.663-3.899 0-6.642 2.038-6.664 4.958-.025 2.16 1.959 3.364 3.451 4.083 1.538.735 2.054 1.207 2.046 1.864-.011 1.006-1.227 1.452-2.359 1.469-1.984.031-3.134-.528-4.051-.948l-.716 3.288c.923.416 2.624.777 4.385.796 4.146-.001 6.856-2.015 6.869-5.133m10.296 4.897h3.647l-3.186-15h-3.364c-.758 0-1.397.432-1.68 1.1l-5.919 13.9h4.142l.822-2.24h5.06l.478 2.24zm-4.402-5.315l2.076-5.633 1.194 5.633h-3.27zm-16.594-9.686l-3.261 15h-3.944l3.261-15h3.944z"
                  fill="#1a1f71"
                  id="Mark_1_"
                />
              </svg>
            </div>
          </div>
          <div>
            <h2
              className="c3"
            >
              Visa
            </h2>
            <span
              className="c8"
            >
              XXXX-XXXX-XXXX-5095
            </span>
            <span
              className="c9"
            >
              $
              220.5
            </span>
            <p
              className="c8"
            >
              Expires 
              04/21
            </p>
          </div>
        </div>
        <section>
          <h2
            className="c3"
          >
            Billing Address
            <button
              className="c10"
              data-testid="modal-trigger"
              onClick={[Function]}
            >
              <svg
                height="18"
                width="18"
                xmlns="http://www.w3.org/2000/svg"
              >
                <circle
                  cx="9"
                  cy="9"
                  fill="#222"
                  r="9"
                />
                <path
                  d="M9.833 6.241H8.167V4.724h1.666v1.517zM8.167 7.49h1.666v5.786H8.167V7.49z"
                  fill="#fff"
                />
              </svg>
            </button>
          </h2>
          <address
            className="c11"
          >
            <p
              className="c12"
            >
              Asd Bgh
            </p>
            <p
              className="c13"
            >
              228 Park Ave S
            </p>
            <p
              className="c13"
            >
              New York, NY 10003
            </p>
            <p
              className="c13"
            >
              (444) 444-4444
            </p>
          </address>
        </section>
        <h2
          className="c3"
        >
          Email Address
        </h2>
        <p
          className="c8"
        >
          asdasdasd@adasad.com
        </p>
      </section>
    </section>
    <section
      className="c14"
    >
      <section
        className="c1"
      >
        <h2
          className="c3"
        >
          Order Summary
        </h2>
        <p
          className="c15"
        >
          <span>
            Subtotal
          </span>
          <span>
            $125.00
          </span>
        </p>
        <p
          className="c16"
        >
          <span>
            Kohl's Cash & Discounts
          </span>
          <span>
            -$18.75
          </span>
        </p>
        <p
          className="c15"
        >
          <span>
            Shipping
          </span>
          <strong>
            FREE
          </strong>
        </p>
        <p
          className="c15"
        >
          <span>
            Tax
          </span>
          <span>
            $6.38
          </span>
        </p>
        <p
          className="c17"
        >
          <span
            className="c18"
          >
            Total
          </span>
          <span
            className="c18"
          >
            $
            122.7
          </span>
        </p>
        <p
          className="c15"
        >
          <span>
            Your Savings
          </span>
          <span>
            $122.70
          </span>
        </p>
      </section>
    </section>
    <div
      className="c19"
    >
      <a
        className="c20"
        data-testid="next-step-button"
        href="/cart"
        onClick={[Function]}
      >
        Place Order
      </a>
    </div>
  </article>,
]
`;
